src/Microsoft.Extensions.Caching.SqlConfig.Tools/Program.cs

```diff
@@ -84,13 +82,15 @@ public int Run(string[] args)
             }
             catch (Exception exception)
             {
-                _logger.LogCritical("An error occurred. {Message}", exception.Message);
+                _logger.LogCritical("An error occurred. {ErrorMessage}", exception.Message);
                 return 1;
             }
         }
 
-        private void CreateTableAndIndexes()
+        private int CreateTableAndIndexes()
         {
+            ValidateConnectionString(_connectionString);
```

Comment:
> Why do you need it? Wouldn't SqlConnection throw if the connection string is invalid?

True, but I did not want to give false positives for any other exception that SqlConnection might throw when its constructing the object, just wanted to be sure that the validation error corresponds to invalid connection string.